go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/secure-systems-lab/go-securesystemslib v0.8.0
 	github.com/segmentio/ksuid v1.0.4
+	github.com/sigstore/protobuf-specs v0.3.0-beta.2
```

Comment:
Can bump up now

Reply:
Done. I also had to use `PublicKeyDetails` as `KnownSignatureAlgorithm` was removed with recent changes.